torch/fx/experimental/GraphManipulation.py

```diff
@@ -0,0 +1,23 @@
+from typing import List
+from torch.fx.graph_module import GraphModule
+from torch.fx.graph import map_arg
+
+
+def get_all_users_of(index: int, fx_module: GraphModule) -> List[int]:
+    "Given the graph(fx_module) and an index, return a list of all node indexes that use this node"
```

Comment:
nit: convention is to use """triple quotes""" for docstrings https://www.python.org/dev/peps/pep-0257/